doc/examples.md

```diff
@@ -39,7 +39,7 @@ storage_dir = pt.io.get_new_storage_dir('my_experiment')
 # storage_dir = pt.io.get_new_subdir('/my/experiment/path')
 ```
 For more information regarding the usage of Configurable or Sacred 
-see [Configurable Introduction]() and [Sacred Introduction]()
+see [Configurable Introduction](configurable) and [Sacred Introduction](sacred)
 
 ### Data preprocessing
 Data preprocessing describes the creation of an iterable object (e.g. a list of examples)for the Padertorch Trainer.
```

Comment:
Comma after e.g., or we should make sure that no one puts a comma after e.g. and i.e..